lib/message.js

```diff
@@ -28,43 +20,49 @@ export default function createMessage(event) {
   embed.setTimestamp(parser.getTime(event));
   embed.setColor(getColor(parser.getLevel(event)));
 
+  const fileLocation = parser.getFileLocation(event);
   embed.setDescription(
-    `\`\`\`${getLanguage(parser.getPlatform(event))}\n${cap(
-      parser.getErrorCodeSnippet(event),
-      4000
-    )}
+    `${fileLocation ? `\`📄 ${fileLocation}\`\n` : ''}\`\`\`${
+      parser.getLanguage(event) || parser.getPlatform(event)
+    }\n${cap(parser.getErrorCodeSnippet(event), 3900)}
     \`\`\``
   );
 
   const location = parser.getErrorLocation(event, 7);
-  embed.addField('Location', cap(location, 1024), true);
+  embed.addField('Stack', `\`\`\`${cap(location.join('\n'), 1000)}\n\`\`\``);
+
+  const tags = parser.getTags(event);
+  if (Object.keys(tags).length > 0) {
+    embed.addField(
+      'Tags',
+      cap(tags.map(([key, value]) => `${key}: ${value}`).join('\n'), 1024),
+      true
+    );
+  }
 
   const user = parser.getUser(event);
   if (user?.username) {
-    embed.addField('**User**', cap(user.username, 1024), true);
+    embed.addField(
+      'User',
+      cap(`${user.username} ${user.id ? `(${user.id})` : ''}`, 1024),
+      true
+    );
   }
 
   const contexts = parser.getContexts(event);
   if (contexts.length > 0) {
-    embed.addField('Contexts', cap(contexts.join('\n'), 1024));
-  }
-
-  const extras = parser.getExtras(event);
-  if (extras.length > 0) {
-    embed.addField('Extras', cap(extras.join('\n'), 1024));
+    embed.addField('Contexts', cap(contexts.join('\n'), 1024), true);
   }
 
   const release = parser.getRelease(event);
-  if (release != null) {
-    embed.addField('Release', cap(release, 1024), true);
```

Comment:
Releases can be tagged whatever the user wants, so I think we want to keep this capped

Reply:
oops